runelite-client/src/main/java/net/runelite/client/game/ItemVariationMapping.java

```diff
@@ -75,4 +86,15 @@ public static int map(int itemId)
 	{
 		return MAPPINGS.getOrDefault(itemId, itemId);
 	}
+
+	/**
+	 * Get item ids for provided variation item id.
+	 *
+	 * @param itemId the item id
+	 * @return the item ids
+	 */
+	public static Collection<Integer> getVariations(int itemId)
+	{
+		return INVERTED_MAPPINGS.asMap().getOrDefault(itemId, Collections.singletonList(itemId));
```

Comment:
Isn't this equivalent to:
```suggestion
		return INVERTED_MAPPINGS.get(itemId);
```

Reply:
The .orDefault is probably important to return original item id in case no variation is found